server/ansible/deploy.yaml

```diff
@@ -0,0 +1,44 @@
+- name: Deploy updates to Draft
+  hosts: all
+  tasks:
+
+    - name: Pull new commits from GitHub
+      git:
+        repo: 'https://github.com/private-attribution/draft.git'
+        dest: '{{ ansible_env.HOME }}/draft'
+        update: yes
+
+    - name: Install packages based on package-lock.json via npm
+      npm:
+        path: '{{ ansible_env.HOME}}/draft/server'
+        state: present
+        ci: true
+
+    - name: Load .env file
+      shell: >
+        aws secretsmanager get-secret-value
+        --secret-id {{ env_secret_id }}
+        --region {{ aws_region }}
+        --query SecretString
+        --output text |
```

Comment:
isn't json the default format and more convenient to use with `jq`?

Reply:
It seems the only difference is if it's a JSON string (e.g., `"{\"KEY\": \"value\"}"` or parsed JSON, (e.g., `{"KEY": "value"}`. It's easy enough to change the output as json (it just require updating the `jq` command to `jq -r 'fromjson | to_entries ...`